config.json

```diff
@@ -832,9 +832,13 @@
       "slug": "robot-simulator",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 5,
```

Comment:
Ok, wasn't sure with that. When I start seeing channels and stuff I tend to think at least 5+ unless it is utterly trivial. It is one of those unique Go things that trips people up.